### DISCUSSION TITLE
Can I change the bandwidth for different frequency on ray tracing?

### DISCUSSION THREAD

**Discussion by Howard-Rhee** [NVlabs/sionna#305] [Q&A] [answered]
Hi!! I am investigating coverage areas based on frequency. I am comparing 4.7GHz and 28GHz and would like to change the bandwidth accordingly and would like to know if there is a way to change the bandwidth. Could you please tell me about this information? Thank you for your nice Ray tracing tool!!

P.S Can Sionna Raytracing can show transmisson??

**Comment by jhoydis** [comment, marked as answer]
Hi,

The coverage map is essentially a path loss map which is independent of the bandwidth used. 

If you want to show something like the received SNR which is bandwidth-dependent, you could compute it like $SNR_\text{dB}=P_\text{dBm} + PL_\text{dB} - 10\log_{10}(WN_o)$,
where $P_\text{dBm}$ is the transmit power in dBm, $PL_\text{dB}$ is the path loss (or coverage map) in dB, $W$ is the bandwidth and $N_0$ is the [noise power spectral density](https://en.wikipedia.org/wiki/Noise_spectral_density).

You can access the path loss in linear scale via [``cm.as_tensor()``](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.CoverageMap.as_tensor).

P.s.:
I assume that you want to know if Sionna RT handles refraction, i.e., transmissions through objects. The answer is no for the moment.

**Reply by Howard-Rhee** [reply]
Thank you for detailed descripition. I understand how to deal my problem.
